renovate.json

```diff
@@ -23,6 +23,17 @@
       "depNameTemplate": "openssl/openssl",
       "versioningTemplate": "regex:^openssl-(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)$",
       "registryUrlTemplate": "https://github.com"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["^scripts/VERSIONS$"],
+      "matchStrings": [
+        "QUICTLS_VERSION=(?<currentValue>.*)\\s"
+      ],
+      "datasourceTemplate": "github-tags",
+      "depNameTemplate": "openssl/openssl",
```

Comment:
I think this actually wants to be quictls/openssl

Reply:
good catch! I'll get that changed